fetch.bs

```diff
@@ -1465,32 +1465,19 @@ null), run these steps. <var>processBody</var> must be an algorithm accepting a
  <li><p>If <var>taskDestination</var> is null, then set <var>taskDestination</var> to the result of
  <a>starting a new parallel queue</a>.
 
- <li><p>Let <var>promise</var> be the result of <a>fully reading body as promise</a> given
- <var>body</var>.
-
- <li><p>Let <var>fulfilledSteps</var> given a <a>byte sequence</a> <var>bytes</var> be to
+ <li><p>Let <var>successSteps</var> given a <a>byte sequence</a> <var>bytes</var> be to
  <a>queue a fetch task</a> to run <var>processBody</var> given <var>bytes</var>, with
  <var>taskDestination</var>.
 
- <li><p>Let <var>rejectedSteps</var> be to <a>queue a fetch task</a> to run
+ <li><p>Let <var>errorSteps</var> be to <a>queue a fetch task</a> to run
  <var>processBodyError</var>, with <var>taskDestination</var>.
 
- <li><p><a for=promise>React</a> to <var>promise</var> with <var>fulfilledSteps</var> and
- <var>rejectedSteps</var>.
-</ol>
-</div>
-
-<div algorithm>
-<p>To <dfn export lt="fully reading body as promise">fully read body as promise</dfn>, given a
```

Comment:
It seems nobody used this so this is safe: https://dontcallmedom.github.io/webdex/f.html#fully%20reading%20body%20as%20promise%40%40fetch%25%25dfn.